Index: smuxi-stable/lib/db4o-net/Db4objects.Db4o/Db4objects.Db4o-2008.csproj
===================================================================
--- smuxi-stable.orig/lib/db4o-net/Db4objects.Db4o/Db4objects.Db4o-2008.csproj	2015-05-22 20:06:38.965459016 +0200
+++ smuxi-stable/lib/db4o-net/Db4objects.Db4o/Db4objects.Db4o-2008.csproj	2015-05-22 20:07:01.973317359 +0200
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build" ToolsVersion="4.0">
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
@@ -10,7 +10,6 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Db4objects.Db4o</RootNamespace>
     <AssemblyName>Db4objects.Db4o</AssemblyName>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     
     
@@ -61,9 +60,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System"/>
-    <Reference Include="System.Core">
-      <RequiredTargetFramework>3.5</RequiredTargetFramework>
-    </Reference>
+    <Reference Include="System.Core"/>
     <Reference Include="System.Data"/>
     <Reference Include="System.Xml"/>
   </ItemGroup>
