Index: smuxi/lib/Twitterizer/Twitterizer2/Twitterizer2.csproj
===================================================================
--- smuxi.orig/lib/Twitterizer/Twitterizer2/Twitterizer2.csproj	2012-01-28 13:26:26.000000000 +0100
+++ smuxi/lib/Twitterizer/Twitterizer2/Twitterizer2.csproj	2012-03-26 12:10:32.000000000 +0200
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
