Index: smuxi/lib/Newtonsoft.Json/Src/Newtonsoft.Json/Newtonsoft.Json.csproj
===================================================================
--- smuxi.orig/lib/Newtonsoft.Json/Src/Newtonsoft.Json/Newtonsoft.Json.csproj	2015-01-11 23:03:43.089234125 +0100
+++ smuxi/lib/Newtonsoft.Json/Src/Newtonsoft.Json/Newtonsoft.Json.csproj	2015-01-11 23:03:43.085234143 +0100
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
@@ -272,4 +272,4 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
-</Project>
\ No newline at end of file
+</Project>
