Index: smuxi/lib/Newtonsoft.Json/Src/Newtonsoft.Json/Newtonsoft.Json.csproj
===================================================================
--- smuxi.orig/lib/Newtonsoft.Json/Src/Newtonsoft.Json/Newtonsoft.Json.csproj	2011-06-26 02:31:01.000000000 +0200
+++ smuxi/lib/Newtonsoft.Json/Src/Newtonsoft.Json/Newtonsoft.Json.csproj	2011-06-26 02:31:33.000000000 +0200
@@ -1,4 +1,4 @@
-﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="3.5">
+﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
@@ -40,7 +40,6 @@
     <IsWebBootstrapper>false</IsWebBootstrapper>
     <UseApplicationTrust>false</UseApplicationTrust>
     <BootstrapperEnabled>true</BootstrapperEnabled>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
